src/app/public/matchers/matchers.spec.ts

```diff
@@ -23,6 +23,7 @@ import 'rxjs/add/observable/of';
 import {
   expect
 } from './matchers';
+import { SkyA11yAnalyzerConfig } from '..';
```

Comment:
One other minor code style thing: This import should be multi-line and appear before the `import` before it since it's "farther away" than the import from a sibling file.

```
import { 
  SkyA11yAnalyzerConfig 
} from '..';

import {
  expect
} from './matchers';
```

Reply:
Fixed and referenced the more specific `../a11y` import path.